### PR TITLE
server: minor log info corrections

### PR DESCRIPTION
**Pull request by mccxj**
log is about listening addr, even though during the unix socket case.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -134,17 +134,20 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
...
 	if cfg.Socket != "" {
 		cfg.SkipAuth = true
-		s.listener, err = net.Listen("unix", cfg.Socket)
+		if s.listener, err = net.Listen("unix", cfg.Socket); err == nil {
+			log.Infof("Server run MySQL Protocol Through Socket [%s]", cfg.Socket)
+		}
 	} else {
-		s.listener, err = net.Listen("tcp", s.cfg.Addr)
+		if s.listener, err = net.Listen("tcp", s.cfg.Addr); err == nil {
+			log.Infof("Server run MySQL Protocol Listen at [%s]", s.cfg.Addr)
```
Comment (ngaut) [inline server/server.go]: s/Server run MySQL Protocol Listen at/Server is running MySQL Protocol at/

File: server/server.go [hunk 2]
```diff
@@ -134,17 +134,20 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 	var err error
 	if cfg.Socket != "" {
 		cfg.SkipAuth = true
-		s.listener, err = net.Listen("unix", cfg.Socket)
+		if s.listener, err = net.Listen("unix", cfg.Socket); err == nil {
+			log.Infof("Server run MySQL Protocol Through Socket [%s]", cfg.Socket)
```
Comment (ngaut) [inline server/server.go]: s/run/is running/

File: server/server.go (hunk 1 above)
Comment (mccxj) [inline server/server.go]: OK, I have modified it.

File: server/server.go (hunk 2 above)
Comment (mccxj) [inline server/server.go]: OK, I have modified it.

File: server/server.go [hunk 3]
```diff
@@ -134,17 +134,20 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 	var err error
 	if cfg.Socket != "" {
 		cfg.SkipAuth = true
-		s.listener, err = net.Listen("unix", cfg.Socket)
+		if s.listener, err = net.Listen("unix", cfg.Socket); err == nil {
+			log.Infof("Server is running MySQL Protocol Through Socket [%s]", cfg.Socket)
```
Comment (shenli) [inline server/server.go]: Through -> through
Rest LGTM

**Comment by shenli** [comment]
@mccxj Thanks!
@ngaut PTAL

**Comment by ngaut** [comment]
LGTM

File: server/server.go (hunk 3 above)
Comment (mccxj) [reply server/server.go]: ok.done.

**Comment by shenli** [comment]
LGTM

**Pull request merged**